python-sdk/tests_integration/databricks_tests/test_load.py

```diff
@@ -84,6 +84,37 @@ def test_delta_load_file_gcs(database_table_fixture):
     database.drop_table(table)
 
 
+@pytest.mark.integration
+@pytest.mark.parametrize(
+    "remote_files_fixture",
+    [
+        {"provider": "azure", "file_create": False},
+    ],
+    indirect=True,
+    ids=["azure_blob_storage"],
+)
+@pytest.mark.integration
+@pytest.mark.parametrize(
+    "database_table_fixture",
+    [
+        {
+            "database": Database.DELTA,
+        }
+    ],
+    indirect=True,
+    ids=["delta"],
+)
+def test_delta_load_file_azure_wasb(database_table_fixture, remote_files_fixture):
```

Comment:
It would be great if we unified the Databricks tests with the other databases' tests, where possible.